src/Text/LLVM/AST.hs

```diff
@@ -480,11 +480,11 @@ isFloatingPoint :: PrimType -> Bool
 isFloatingPoint (FloatType _) = True
 isFloatingPoint _             = False
 
-isAlias :: Type -> Bool
+isAlias :: Type' ident -> Bool
 isAlias Alias{} = True
 isAlias _       = False
 
-isPrimTypeOf :: (PrimType -> Bool) -> Type -> Bool
+isPrimTypeOf :: (PrimType -> Bool) -> Type' lab -> Bool
```

Comment:
The code here is inconsistent about calling `Type`'s type parameter `ident` or `lab` (and similarly with the code below). I think we should pick one convention and stick to it.